### ISSUE TITLE
idk if you are working on this add Support  for the hbm nt mod 1.12.2 it is a some what of a big mod

### ISSUE THREAD

**Issue by madcan1** [TagnumElite/ProjectE-Integration#95]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by TagnumElite** [comment]
I have looked into it. When I look at the source code, everything is hard coded. I have use that but I don't like it. I you really want that I probably can add it in a few days.

**Comment by festro** [comment]
Yeah HBM has practically been a new fav in my 1.12 modpacks, so compatibility would be great.
